### ISSUE TITLE
change icons in navigation bar

### ISSUE THREAD

**Issue by LAKnoKAL** [zippyui/react-date-picker#163]
Hey, guys! So the question is pretty simple - Can I change icons in the navigation bar? In my case I need to change it on font icons, so basically will be great to just add the class on the element and remove SVG element.

I understand that I can do it manually by hiding SVG element and add styles that applying on icon classes but I'm sure that it's not a good solution because in this case need to duplicate some styles.

Thnx for help!

**Comment by dev-zippyui** [comment]
You can use `navBarArrows` prop on the `DatePicker`

```jsx
const navBarArrows = {
  'prev': '<', //any valid jsx
  'next': '>' //any valid jsx
}
<DatePicker
  navBarArrows={navBarArrows}
/>
```

or, if you want to control also the << and >>, just use
```js
const navBarArrows = {
  '-1': '<', //any valid jsx
  '1': '>', //any valid jsx
  '-2': '<<',
  '2: '>>'
}
```

**Comment by LAKnoKAL** [comment]
@dev-zippyui wow! Pretty awesome! Is it info in docs?

**Comment by dev-zippyui** [comment]
No, we don't have readme docs for this just yet, just examples showing the configuration applied. See http://zippyui.com/docs/react-date-picker/examples

**Comment by LAKnoKAL** [comment]
@dev-zippyui yeah, but there is no info about it. So maybe we can qualify this task to add this info to the docs in case if you don't have this one?
